media/js/firefox/new.js

```diff
@@ -148,167 +146,114 @@
-                // still need to use the regular track_and_redirect() function because
+                // still need to use the regular trackAndRedirect() function because
                 // the popup will be blocked and then the download will also be blocked
                 // in the popup.
                 if (window.location.hash === '#download-fx' || params.get('scene') === 2) {
-                    track_and_redirect(url, virtual_url);
+                    trackAndRedirect(url, virtualUrl);
                 } else {
-                    track_and_popup(url, virtual_url);
+                    trackAndPopup(url, virtualUrl);
                 }
             } else {
-                track_and_redirect(url, virtual_url);
+                trackAndRedirect(url, virtualUrl);
             }
 
-            if ($stage.data('scene') !== 2) {
-                if (hash_change) {
+            if ($downloadInteraction.data('scene') !== 2) {
+                if (hashChange) {
                     window.location.hash = '#download-fx';
                 } else {
-                    show_scene_anim(2);
+                    showScene(2);
                 }
             }
         });
 
-        if (hash_change && !no_scene2) {
+        if (hashChange && !noScene2) {
             $window.on('hashchange', function() {
                 if (window.location.hash === '#download-fx') {
-                    show_scene_anim(2);
+                    showScene(2);
                 }
                 if (window.location.hash === '') {
-                    show_scene_anim(1);
+                    showScene(1);
                 }
             });
         }
 
         $window.on('load', function() {
-            // Add CSS class that allows scene2 images to load. Done on ready()
-            // so as not to block the loading of other images.
-            $('body').addClass('ready-for-scene2');
-
             // initiate download/scene2 if coming directly to #download-fx and/or ?scene=2
             // some older browsers will not preserve the #download-fx when they are redirected
             // so we use a url parameter, but we don't want to use a url param when the user
             // clicks the download button on /firefox/new/ because that can trigger an unecessary page load
             if (window.location.hash === '#download-fx' || params.get('scene') === 2) {
-                if (no_scene2) {
+                if (noScene2) {
                     // if using an unsupported platform just try to drop the URL hash
                     if (window.history && window.history.replaceState) {
                         var uri = window.location.href.split('#')[0];
                         window.history.replaceState({}, '', uri);
                     }
                 } else {
-                    show_scene(2);
+                    showScene(2);
                     // For IE < 11 we supress the auto-download since this
```

Comment:
We should totally make a bug to do this now that the PHP site is no more? Needn't be part of this PR, but we *should* be able to now limit the popup behavior to IE 6/7/8?